concordium-contracts-common/src/schema.rs

```diff
@@ -333,6 +290,8 @@ pub enum Type {
     ByteList(SizeLength),
     /// A fixed sized list of bytes.
     ByteArray(u32),
+    /// An enum with a tag.
```

Comment:
I don't think this type is very odd. In fact the Enum one is more odd, since it hardcodes the tags to be the indices in the list, which is ad-hoc. This is the more "obvious" type.

I would also not say that this is anything to do with events. This can be used as any other type. It makes sense for a parameter as well.

Reply:
I see your point. Okay, it's not that odd then ;)